superset-frontend/plugins/legacy-preset-chart-nvd3/src/Pie/index.js

```diff
@@ -24,11 +24,15 @@ import controlPanel from './controlPanel';
 const metadata = new ChartMetadata({
   credits: ['http://nvd3.org'],
   description: '',
-  name: t('Pie Chart'),
+  name: t('Pie Chart (deprecated)'),
```

Comment:
Hmm, I wonder if it's fair to add `deprecated: true` to the metadata here, which would hide it from the viz gallery. Maybe we should gather some thoughts on this at Town Hall tomorrow? I'll stick it on the agenda.

Reply:
This one does not appear in the viz gallery already. I just marked it as deprecated to be removed in 4.0.